gui/default/index.html

```diff
@@ -330,9 +330,67 @@ <h3 class="panel-title">
         </div>
       </div>
 
-      <!-- First regular row -->
+      <!-- Login form -->
+      <div class="center-block" ng-if="!authenticated">
+        <h3 translate>Authentication required</h1>
 
-      <div class="row">
+        <form ng-submit="authenticatePassword()" class="col-md-6">
+          <div class="row form-group">
+            <label translate for="Username">Username</label>
+            <input id="Username" class="form-control" type="text" ng-model="login.username" tabindex="1" />
+          </div>
+
+          <div class="row form-group">
+            <label translate for="Password">Password</label>
+            <input id="Password" class="form-control" type="password" ng-model="login.password" ng-trim="false" tabindex="2" />
+          </div>
+
+          <div class="row">
+            <div class="col-md-9">
+              <p class="text-danger" ng-if="login.errors.badLogin">
+                Incorrect username and/or password.
+              </p>
+              <p class="text-danger" ng-if="login.errors.failed">
+                Login failed, see server log for details.
+              </p>
+            </div>
+            <div class="col-md-3" style="text-align: right; padding-right: 0">
+              <button type="submit" class="btn btn-default" ng-disabled="login.inProgress" translate>Log in</button>
+            </div>
+          </div>
+        </form>
+
+        <div class="col-md-6" style="text-align: center">
+          <p translate>
+            Or...
+          </p>
+          <button type="button" class="btn btn-primary" ng-click="authenticateWebauthn()" ng-disabled="login.inProgress || isRawIpAddress() || getLocationProtocol() !== 'https'" tabindex="3">
+            <span translate>Log in with WebAuthn</span>
+          </button>
+
+          <p class="text-info" ng-if="isRawIpAddress()" translate>
+            WebAuthn is not supported on raw IP address hosts.
+          </p>
+          <p class="text-info" ng-if="getLocationProtocol() !== 'https'" translate>
+            WebAuthn requires HTTPS.
+          </p>
+          <p class="text-danger" ng-if="webauthn.errors.notRegistered" translate>
+            This device or security key is not registered. Please try a different one.
+          </p>
+          <p class="text-info" ng-if="webauthn.errors.aborted" translate>
+            Authentication aborted by user.
+          </p>
+          <p class="text-danger" ng-if="webauthn.errors.notAllowed" translate>
+            Authentication failed - possibly aborted, timed out, or blocked by the browser.
+          </p>
+          <p class="text-danger" ng-if="webauthn.errors.authenticationFailed" translate>
+            Authentication failed, see server log for details.
```

Comment:
What "server" is being referred here specifically?

Reply:
The Syncthing background service. I've also been thinking this should perhaps make less of a promise that more information will actually be available. How about this?

```suggestion
            Authentication failed. More information may be available in the Syncthing service logs or the browser's developer console.
```

Reply:
How about just "in the Syncthing log"?

Reply:
Thanks, done now.